packages/web-component/src/lib/descope-wc/DescopeWc.ts

```diff
@@ -595,6 +654,17 @@ class DescopeWc extends BaseDescopeWc {
       await this.#handleWebauthnConditionalUi(clone, next);
     }
 
+    if (
+      currentState.samlIdpUsername &&
+      !screenState.form?.loginId &&
+      !screenState.form?.email
+    ) {
+      screenState.form = {
+        loginId: currentState.samlIdpUsername,
+        email: currentState.samlIdpUsername,
+      };
+    }
```

Comment:
Can we do this instead?

```
screenState.form.loginId ??= currentState.samlIdpUsername;
screenState.form.email ??= currentState.samlIdpUsername;

```

Reply:
@orius123 ??
(this is the code that set the username we get from Azure)

Reply:
This is for the form we send immediately, if so or doesn't matter, if it is for actual screens we need to append like Nor said

Reply:
ok cool so Ill push Nir suggestion..

Reply:
@orius123 but are we ok with replacing all this code block with Nirs two lines? as I think we still want to keep the "if" inplace.. am I right?

Reply:
yes